pwndbg/commands/context.py

```diff
@@ -115,10 +124,12 @@ def context(*args):
 def context_regs():
     return [pwndbg.ui.banner("registers")] + get_regs()
 
-
-@pwndbg.commands.Command
+parser = argparse.ArgumentParser()
+parser.description = '''Print out all registers and enhance the information.'''
```

Comment:
```suggestion
parser.description = 'Print out all registers and enhance the information.'
```

Reply:
Looks like a good chance, but for future reference, how much does this matter?
Is it just better style to use one quote if the string is just one line?

Reply:
> how much does this matter?
It doesn't; tho it is good to keep a proper coding style.

If you haven't seen pep8 it's probably good to read that https://www.python.org/dev/peps/pep-0008/ (and also take some of the rules with a grain of salt).